src/baseline_bots/randomize_dipnet.py

```diff
@@ -0,0 +1,234 @@
+"""
+Some functions that generate randomized orders from 
+an already existing order / list of orders.
+"""
+
+__author__ = "Konstantine Kahadze"
+__email__ = "konstantinekahadze@gmail.com"
+
+import random
+from typing import List
+
+# the comments below signal the formatter not to expand these dicts to multiple lines
```

Comment:
Sorry, I meant comments to describe ADJACENCY, TYPES, and COMBOS. What is the point of each of these constants?